<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Model_Evaluate_2</class>
 <widget class="QMainWindow" name="Model_Evaluate_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluate Window</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="transform_sqrtSign">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform sqrt</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="BinSizeSign">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Bin Size</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="BlockNormSign">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Block Norm</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutputsSign">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Outputs</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaxImageSign">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Max Images</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="DescriptorSign">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Descriptor</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="transform_sqrt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>28</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="BinSize">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="CellSizeSign">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>124</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Cell Size</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="CellSize">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QComboBox" name="BlockNorm">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>144</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="MaxImage">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="SaveDescriptor">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>123</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Process">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>261</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Process Images</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="Select">
    <property name="title">
     <string>Select Page</string>
    </property>
   </widget>
   <addaction name="Select"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
